Backend/app/provider-platform/dynamic-offer-driver-app/Main/src-read-only/Storage/Beam/Client.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module Storage.Beam.Client where
+
+import qualified Database.Beam as B
+import Kernel.External.Encryption
+import Kernel.Prelude
+import qualified Kernel.Prelude
+import Tools.Beam.UtilsTH
+
+data ClientT f = ClientT
+  { id :: (B.C f Kernel.Prelude.Text),
```

Comment:
for some reason TH adds redundant brackets. For removing this brackets please use command
`, run-generator  -- --apply-hint`